### PR TITLE
MessageBuilder setters entirely fluent

### PR DESCRIPTION
**Pull request by finwe**
BC Breaks - to be merged to 3.0

### CODE REVIEW

File: src/Mailgun/Message/MessageBuilder.php [hunk 1]
```diff
@@ -467,6 +465,14 @@ public function setDeliveryTime($timeDate, $timeZone = null)
         $formattedTimeDate = $dateTimeObj->format(\DateTime::RFC2822);
         $this->message['o:deliverytime'] = $formattedTimeDate;
 
+        return $this;
+    }
+
+    /**
+     * @return string
+     */
+    public function getDeliveryTime()
```
[the remark this answers is not in the document]
Comment (DavidGarciaCat) [reply src/Mailgun/Message/MessageBuilder.php]: Agree on this one

File: src/Mailgun/Message/MessageBuilder.php [hunk 2]
```diff
@@ -499,6 +503,16 @@ public function addCustomParameter($parameterName, $data)
...
 
+        return $this;
+    }
+
+    /**
+     * @param string $parameterName
+     *
+     * @return mixed
+     */
+    public function getCustomParameter($parameterName)
```
[the remark this answers is not in the document]
Comment (DavidGarciaCat) [reply src/Mailgun/Message/MessageBuilder.php]: Agree on this one

File: src/Mailgun/Message/MessageBuilder.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (finwe) [reply src/Mailgun/Message/MessageBuilder.php]: Done.

File: src/Mailgun/Message/MessageBuilder.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (finwe) [reply src/Mailgun/Message/MessageBuilder.php]: Done.

**Comment by Nyholm** [comment]
I see that you just did a rebase but something does not look right. Will you try again?

**Comment by finwe** [comment]
I did the rebase now. The diff seem OK, right?

**Comment by Nyholm** [comment]
Yes, perfect. Thanks!

**Pull request merged**